cmd/contour/shutdownmanager.go

```diff
@@ -38,14 +38,17 @@ const (
 
 // File path used in the /shutdown endpoint.
 const shutdownReadyFile = "/ok"
+const shutdownReadyCheckInterval = time.Second * 1
```

Comment:
nit: since we have a comment on the previous constant, it'd be nice to add one for this new const as well.

Reply:
Done.